### PR TITLE
timeconverter

### PR DESCRIPTION
**Pull request by rabibasukala01**

### CODE REVIEW

File: C-Programs/time-convert.c [hunk 1]
```diff
@@ -0,0 +1,22 @@
+//convert seconds to hour,minute and second
+#include<stdio.h>
+
+int main(void)
+{
+    long minute,hour,second,inputsec,hourRemainder,minuteRemainder;
+   
+    printf("enter numbers in seconds ");
+    scanf("%d",&inputsec);
```
Comment (PrajwalCH) [inline C-Programs/time-convert.c:9]: Use **%ld** here for long data type.

**Comment by rabibasukala01** [comment]
i guess its %lu for unsigned ?

**Comment by PrajwalCH** [comment]
> i guess its %lu for unsigned ?

Yes. But long is not a unsigned it's signed. For unsigned long You have to explicitly declare **unsigned long**.

**Review by PrajwalCH — APPROVED** [review]

**Review by rabibasukala01 — COMMENTED** [review]
done

**Pull request merged**